cmd/dlv/dlv_test.go

```diff
@@ -1020,6 +1020,10 @@ func TestTrace(t *testing.T) {
 }
 
 func TestTraceMultipleGoroutines(t *testing.T) {
+	if runtime.GOOS == "freebsd" {
+		t.Skip("broken")
```

Comment:
Let's change this to something more like "temporarily disabled due to issues with FreeBSD in Delve and Go".

Reply:
Let's also add a TODO to investigate further.

Reply:
Done.